charts/civil-ccd/values.enableWA.preview.template.yaml

```diff
@@ -1,9 +1,36 @@
-civil-wa:
+servicebus:
```

Comment:
This whole file doesn't match master. Here we have Camunda bpmn as false thats why your PR is failing. Please make it same as master. I dont believe as part of this ticket you need to make any changes in this yaml